parkings/api/operator/permit.py

```diff
@@ -0,0 +1,92 @@
+from django.db import transaction
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import mixins, serializers
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from parkings.models import EnforcementDomain, Permit, PermitSeries
+
+from ..common_permit import (
+    ActivePermitByExternalIdSerializer, ActivePermitByExternalIdViewSet,
+    PermitSerializer, PermitSeriesViewSet, PermitViewSet)
+from .permissions import IsOperator
+
+
+class OperatorPermitSeriesViewSet(mixins.DestroyModelMixin, PermitSeriesViewSet):
+    permission_classes = [IsOperator]
+
+    @action(detail=True, methods=['post'])
+    def activate(self, request, pk=None):
+        with transaction.atomic():
+            obj_to_activate = self.get_object()
+            old_actives = self.queryset.filter(active=True, owner=self.request.user)
+
+            # Always activate the specified permit series regardless of what to deactivate
+            if not obj_to_activate.active:
+                obj_to_activate.active = True
+                obj_to_activate.save()
+
+            deactivate_payload = OperatorPermitSeriesPayload(data=request.data)
+            deactivate_payload.is_valid(raise_exception=True)
+            validated_data = deactivate_payload.validated_data
```

Comment:
This body data validation should be done first, even before starting the transaction.